src/display/font_loader.js

```diff
@@ -365,9 +381,33 @@ var FontFaceObject = (function FontFaceObjectClosure() {
 
     getPathGenerator: function FontLoader_getPathGenerator(objs, character) {
       if (!(character in this.compiledGlyphs)) {
-        var js = objs.get(this.loadedName + '_path_' + character);
-        /*jshint -W054 */
-        this.compiledGlyphs[character] = new Function('c', 'size', js);
+        var cmds = objs.get(this.loadedName + '_path_' + character);
+        var js = '';
+        var current, i, len, j, alen;
```

Comment:
`j` and `alen` don't seem to be used, so remove them.

And could you move `var js = '';` inside `if (FontLoader.isEvalSupported) {`?